docs/CONTRIBUTING.md

```diff
@@ -16,10 +16,11 @@ There are other branches as well on Quick Pad, these are usually for new feature
 
 ## Contributing to Quick Pad
 
-To get started, create a fork of the repository on GitHub. You will need to do that to make any changes since the Quick Pad repo is restricted to a few contributors of the app. If you decide to make a change you should always do it on the `develop` branch. Once your change is completed you can open a pull request which we will review and if we like it we will merge it in 😀
+To get started, create a fork of the repository on GitHub. You will need to do that to make any changes since the Quick Pad repo is restricted to a few contributors of the app. If you decide to make a change you should always do it on the `develop` branch. Once your change is completed you can open a pull request which we will review and if we like it we will merge it in 
```

Comment:
Please add a period at the end of the sentence.

Reply:
ok